docs/fsharp/language-reference/functions/recursive-functions-the-rec-keyword.md

```diff
@@ -70,6 +87,14 @@ The following example shows two mutually recursive functions.
 
 [!code-fsharp[Main](~/samples/snippets/fsharp/lang-ref-1/snippet4002.fs)]
 
+## Recursive values
+
+You can also define a `let`-bound value to be recursive. This is sometimes done for logging. With F# 5 and the `nameof` function, you can do this:
```

Comment:
In C#, `nameof` is an operator not a function. I don't know about F#, but just wanted to note.

Reply:
The distinction between operators and functions in F# isn't quite so clear. But it's defined as a function.